### PR TITLE
TEST hbase stuff

### PR DESCRIPTION
**Pull request by dhalperi**
cc: @iemejia

### CODE REVIEW

**Comment by dhalperi** [comment]
Looks like it got farther -- master sets up, but the regionserver is what fails.

**Comment by dhalperi** [comment]
Closing, now that I got the HBaseIOTest to pass (https://builds.apache.org/job/beam_PreCommit_Java_MavenInstall/7544/org.apache.beam$beam-sdks-java-io-hbase/testReport/org.apache.beam.sdk.io.hbase/HBaseIOTest/)

File: sdks/java/io/hbase/src/test/java/org/apache/beam/sdk/io/hbase/HBaseIOTest.java [hunk 1]
```diff
@@ -98,7 +98,8 @@
     public static void beforeClass() throws Exception {
         conf.setInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER, 1);
         // Try to bind to localhost (this is apparently only for the web console)
```
Comment (dhalperi) [inline sdks/java/io/hbase/src/test/java/org/apache/beam/sdk/io/hbase/HBaseIOTest.java]: nothing you may want to remove some of the other comments/test hacks around this if these two properties do indeed do the right trick.

**Comment by iemejia** [comment]
Wow, thanks a lot for finding the issue, that was great !!!
I will fix and PR to see if the full build passes now, excellent!

**Pull request closed without merging**